setup.py

```diff
@@ -351,6 +351,8 @@ def get_extensions():
 
     ffmpeg_exe = distutils.spawn.find_executable('ffmpeg')
     has_ffmpeg = ffmpeg_exe is not None
+    if sys.platform != 'linux':
+        has_ffmpeg = False
```

Comment:
For my info, do we need to disable also for Windows?

Reply:
I tried it on Windows once and saw some failure. After fixing that, I'll re-enable it here.